backend/database/models/user.py

```diff
@@ -44,8 +44,13 @@ def compile_ci_string(element, compiler, **kwargs):
         return base_visit
 
 
+class UserRole(enum.Enum):
```

Comment:
Darrell also listed contributor and admin roles in #104

Reply:
Added!